### PR TITLE
SDK examples

### PR DESCRIPTION
**Pull request by OvidijusSobut**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -19,10 +45,210 @@ This library is built atop of [PSR-7](https://www.php-fig.org/psr/psr-7/) and
...
+
+<a name="get-a-list-of-partners"></a>
+### Get a list of partners
+
+```php
+use Partnero\Partnero;
+
+$partnero = new Partnero("api_key");
+
+$partnero->partners()->list(limit: 10);
```
Comment (mindaugasmailer) [inline README.md]: Maybe let remove variable names.

File: README.md [hunk 2]
```diff
@@ -19,10 +45,210 @@ This library is built atop of [PSR-7](https://www.php-fig.org/psr/psr-7/) and
...
+```php
+use Partnero\Partnero;
+use Partnero\Models\Partner;
+
+$partnero = new Partnero("api_key");
+
+$partner = (new Partner())
+  ->setEmail('test@mail.com')
+  ->setName("Name")
+  ->setSurname("Surname")
```
Comment (mindaugasmailer) [inline README.md]: Let remove surname too, we will move to use only name in the future

File: README.md [hunk 3]
```diff
@@ -19,10 +45,210 @@ This library is built atop of [PSR-7](https://www.php-fig.org/psr/psr-7/) and
...
+### Update partner
+
+```php
+use Partnero\Partnero;
+use Partnero\Models\Partner;
+
+$partnero = new Partnero("api_key");
+
+$partner = (new Partner())
+  ->setEmail('john.doe@mail.com')
```
Comment (mindaugasmailer) [inline README.md:101]: Maybe we need to be consistent with double or single quotes. I'm voting for single.

File: README.md [hunk 4]
```diff
@@ -19,10 +45,210 @@ This library is built atop of [PSR-7](https://www.php-fig.org/psr/psr-7/) and
...
+```php
+use Partnero\Partnero;
+
+$partnero = new Partnero("api_key");
+
+$partnero->customers()->delete('new-customer-key');
+```
+
+<a name="settings-api"></a>
+## Settings
```
Comment (mindaugasmailer) [inline README.md]: Drop settings part to the end, we need to think if we need this one at all.

**Review by mindaugasmailer — CHANGES_REQUESTED** [review]
I think main thing to be consistent through all readme

File: README.md (hunk 2 above)
Comment (mindaugasmailer) [reply README.md]: hmmm, maybe lets leave it for partners and don't show for customers only 🤔

**Review by gdmns — APPROVED** [review]

**Pull request merged**